### ISSUE TITLE
Specify Alt text for the Search icon on doc landing page

### ISSUE THREAD

**Issue by dedemorton** [elastic/docs#71]
As far as I can tell, the search icon on the doc landing page (https://www.elastic.co/guide/index.html) doesn't include Alt text. The lack of Alt text makes this functionality hard to find for people who are visually impaired.
